portfolio/src/main/java/com/google/sps/servlets/VideoServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Random; 
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that helps avoid getting the same video twice in a row.*/
+@WebServlet("/video")
+public class VideoServlet extends HttpServlet {
+
+  public int lastVideoIndex = 4;
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Can you add some Javadoc here explaining what this method does? We typically want Javadoc on all public methods: https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
Done!